scrapers/urbandictionary.rb

```diff
@@ -21,6 +25,13 @@ def get_definitions(term)
     definitions.map { |m| parse_definition(term, m) }
   end
 
+  def get_random_definition()
```

Comment:
I agree about the `get_`
Can you explain about the `()`? I'm not familiar with this feature